cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,30 @@
-describe('', () => {
-  before(() => {
+/// <reference types='cypress' />
+const { generateUser } = require('../support/generateUser');
+const { generateArticle } = require('../support/generateArticle');
 
+describe('Conduit article editor', () => {
+  beforeEach(() => {
+    const user = generateUser();
+    cy.login(user.email, user.username, user.password);
   });
 
-  it('', () => {
-
+  it('should allow to post an article', () => {
+    const article = generateArticle();
+    cy.createArticle(article.title, article.description, article.body)
+      .then((response) => {
+        cy.visit(`/article/${response.body.article.slug}`);
+      });
+    cy.get('h1').should('contain.text', article.title);
+    cy.get('p').should('contain.text', article.body);
   });
 
-  it('', () => {
-
+  it('should allow to delete posted article', () => {
```

Comment:
add an assertion in the end of this test, that your feed should contain text "There are no article..."